src/Tizen.NUI/src/public/Accessibility/Accessibility.cs

```diff
@@ -94,6 +94,38 @@ public static bool IsScreenReaderEnabled
             }
         }
 
+        /// <summary>
+        /// Gets or sets the enabled state of Screen Reader.
+        /// </summary>
+        /// <remarks>
+        /// Getter returns true if Screen Reader is enabled, false otherwise.
+        /// </remarks>
+        /// This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static bool EnableScreenReader
```

Comment:
다음과 같은 어순의 이름이 좀 더 보편적이고 다른곳과도 유사한 네이밍인것 같습니다.
```suggestion
        public static bool ScreenReaderEnabled
```